src/main/java/seedu/address/model/person/Person.java

```diff
@@ -136,8 +136,8 @@ public String toString() {
                 .append(getPriceRange().map(PriceRange::toString).orElse("Budget: Not Specified"))
                 .append("; Desired Characteristics: ")
                 .append(getDesiredCharacteristics()
-                        .map(DesiredCharacteristics::toString)
-                        .orElse("Desired Characteristics: Not Specified"));
+                        .map(Characteristics::toString)
+                        .orElse("Not Specified"));
```

Comment:
Since characteristics is an optional property, should we still display a "not specified"?

Reply:
Right! I think we should have it not displayed if not specified, but until we get to the FXML i'll leave it like this first?